src/index.html

```diff
@@ -1,13 +1,64 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <!-- <link
+      rel="stylesheet"
+      href="style.css"
+    > -->
+    <link rel="stylesheet" href="./styles/index.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="use-credentials"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <div class="card" data-qa="card">
+      <img
+        src="./images/imac.jpeg"
+        alt="Product image"
+        class="card__img"
+      >
+      <h2 class="card__title big-text">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </h2>
+
+      <p class="card__subtitle small-text">Product code: 195434</p>
+      <div class="reviews">
+        <div class="reviews__stars stars--4 stars">
+          <div class="stars__star"></div>
+
+          <div class="stars__star"></div>
+
+          <div class="stars__star"></div>
+
+          <div class="stars__star"></div>
+
+          <div class="stars__star"></div>
+        </div>
+
+        <p class="reviews__text small-text">Reviews: 5</p>
+      </div>
+
+      <div class="card__payment">
+        <p class="card__payment-subtitle small-text">Price:</p>
+
+        <p class="card__payment-price big-text">$2,199</p>
+      </div>
```

Comment:
I believe, looks better now
```suggestion
      <div class="reviews">
        <div class="reviews__stars stars--4 stars">
          <div class="stars__star"></div>
          <div class="stars__star"></div>
          <div class="stars__star"></div>
          <div class="stars__star"></div>
          <div class="stars__star"></div>
        </div>

        <p class="reviews__text small-text">
          Reviews: 5
        </p>
      </div>

      <div class="card__payment">
        <p class="card__payment-subtitle">
          Price:
        </p>

        <p class="card__payment-price">
          $2,199
        </p>
      </div>
```